_security/fips.md

```diff
@@ -0,0 +1,179 @@
+---
+layout: default
+title: Getting started with OpenSearch FIPS
```

Comment:
Maybe "OpenSearch FIPS configuration"? The "getting started" is only needed when there's much more content to signal to users that this is the basic topic.